Strategy11/formidable-forms
# Add global setting for form submission message

Fix https://github.com/Strategy11/formidable-pro/issues/3739

This PR adds a feature that announces a form submission message with screen reader when a form is submitted. There is also a new global setting added for updating the message.

<img width="1271" alt="image" src="https://github.com/Strategy11/formidable-forms/assets/41271840/d1d624af-c800-4fda-8083-035508641c1f">

### Test steps:
1. Go to the global settings and check that a new setting 'Submit Speak Text' is there.
2. Modify it as needed for your test.
3. Turn on a screen reader and try submitting a form.
4. Confirm that the message is announced by your screen reader.
